xpallets/gateway/common/src/lib.rs

```diff
@@ -337,7 +395,6 @@ impl<T: Config> Module<T> {
 }
 
 pub fn is_valid_about<T: Config>(about: &[u8]) -> DispatchResult {
-    // TODO
```

Comment:
There is no description in this `TODO` and it may not be completed, I am not sure whether it should be removed

Reply:
Maybe you are right, the code exists in the old version of the file, I probably shouldn’t change it.